src/components/Modal/Modal.module.scss

```diff
@@ -0,0 +1,20 @@
+.ModalWrapper {
+  display: none;
+
+  &Active {
+    align-items: center;
+    background: var(--GraySecondary--translucent);
+    display: flex;
+    height: 100vh;
+    justify-content: center;
+    left: 0;
+    position: fixed;
+    top: 0;
+    width: 100vw;
+  }
+}
+
+.ModalContent {
+  position: relative;
+  overflow: scroll;
```

Comment:
What if we don't have enough content to scroll? Should we use `overflow: auto` here instead?

Reply:
Yep, it should be. The interesting thing is I didn't catch this because macOS breaks this behavior by default